extension.neon

```diff
@@ -29,6 +29,7 @@ parameters:
 			-
 				type: Expr_FuncCall
 				functions:
+					- dd
```

Comment:
Laravel's function right?

Reply:
It's the Symfony function provided by the VarDumper Component for call dump(...) then die().
So it's probably in Laravel too.

Reply:
Oh right, could you please amend your commit to update the CHANGELOG.md?

Reply:
Done 👍